pandas/tests/extension/test_boolean.py

```diff
@@ -112,9 +112,9 @@ def _check_op(self, s, op, other, op_name, exc=NotImplementedError):
                 # subtraction for bools raises TypeError (but not yet in 1.13)
                 if _np_version_under1p14:
                     pytest.skip("__sub__ does not yet raise in numpy 1.13")
-                with pytest.raises(TypeError):
+                msg = r"numpy boolean subtract, the \`-\` operator"
```

Comment:
```suggestion
                msg = r"numpy boolean subtract"
```

This is an error message from numpy, so we shouldn't match too much